### PR TITLE
[flang][OpenMP] Follow-up to build-breakage fix

### PR DESCRIPTION
**Pull request by kparzysz**
Adjust the handling of a few of the new clauses.

### CODE REVIEW

**Comment by kparzysz** [comment]
cc: @jtb20

**Comment by kparzysz** [comment]
Sorry, got a bit late to this.  The previous fix is mostly correct---some of the new clauses are empty, so we can have a complete handling of them instead of marking them as "unimplemented".

File: flang/lib/Lower/OpenMP/Clauses.h [hunk 1]
```diff
@@ -177,6 +177,7 @@ using Capture = tomp::clause::CaptureT<TypeTy, IdTy, ExprTy>;
 using Collapse = tomp::clause::CollapseT<TypeTy, IdTy, ExprTy>;
 using Contains = tomp::clause::ContainsT<TypeTy, IdTy, ExprTy>;
 using Compare = tomp::clause::CompareT<TypeTy, IdTy, ExprTy>;
+using Contains = tomp::clause::ContainsT<TypeTy, IdTy, ExprTy>;
```
Comment (jtb20) [inline flang/lib/Lower/OpenMP/Clauses.h:180]: I think you duplicated "Contains" here?

**Comment by tblah** [comment]
Thanks for following up

**Review by skatrak — APPROVED** [review]
LGTM, thanks!

**Pull request merged**